web/cypress/integration/service/save-paginator-status.spec.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-undef */
+
+context('Save Paginator Status', () => {
+  const timeout = 300;
+  beforeEach(() => {
+    cy.login();
+
+    cy.fixture('selector.json').as('domSelector');
+    cy.fixture('data.json').as('data');
+  });
+
+  it('should create 11 test services', function () {
+    cy.visit('/');
+    cy.contains('Service').click();
+
+    for (let i = 0; i <= 10; i++) {
+      cy.contains('Create').click();
```

Comment:
There are already testcase for testing the creation of services through the UI. Using the api call to create the service here will save a lot of time.

Reply:
ping @guoqqqi

Reply:
done